pymc_experimental/statespace/core/statespace.py

```diff
@@ -291,7 +314,7 @@ def _unpack_statespace_with_placeholders(self) -> Tuple:
 
         return a0, P0, c, d, T, Z, R, H, Q
 
-    def unpack_statespace(self) -> Tuple:
+    def unpack_statespace(self) -> list[pt.TensorVariable, ...]:
```

Comment:
List doesn't need ellipsis 

```suggestion
    def unpack_statespace(self) -> list[pt.TensorVariable]:
```

Reply:
Pycharm actually complains when I make this change, it wants me to literally list out `tuple[
        TensorVariable, TensorVariable, TensorVariable, TensorVariable, TensorVariable, TensorVariable, TensorVariable, TensorVariable, TensorVariable]`, which seems silly. `list[TensorVariable, ...]` was my hack-around.

Reply:
Ignore pycharm